os/board/common/README.md

```diff
@@ -0,0 +1,78 @@
+# How to partition a flash
+
+This document provides a way to partition in flash. Use CONFIG when flash partition.  
+
+## How to configure CONFIG
+
+Flash partition is provided by configurations, FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME. 
+FLASH_PART_SIZE represents size of each partition, FLASH_PART_SIZE does name, FLASH_PART_TYPE does interface between flash and upper layer.  
+All partition information is separated by comma and the last must end with a comma. The information of FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME should be in order. 
```

Comment:
The information of FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME should be in order.  --> "Note that the values for FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME must be specified in that order, as shown in the example below:"

Reply:
thanks.